lib/gcloud.rb

```diff
@@ -374,4 +374,41 @@ def logging scope: nil
     require "gcloud/logging"
     Gcloud.logging @project, @keyfile, scope: scope
   end
+
+  ##
+  # Creates a new object for connecting to the Translate service.
+  # Each call creates a new connection.
+  #
+  # TODO: Add info for creatign an API Key here...
+  # TODO: Explain that the API Key is likely temproary and can change in a
+  # future release.
+  #
+  # @param [String] key API Key is blah blah blah...
+  #
+  # @return [Gcloud::Translate::Api]
+  #
+  # @example
+  #   require "gcloud"
+  #
+  #   gcloud = Gcloud.new
+  #   translate = gcloud.translate "api-key-abc123XYZ789"
+  #
+  #   translation = translate.translate "Hello world!", to: "la"
+  #   puts translation #=> Salve mundi!
```

Comment:
If you wanted to show the quotes on the string the I would replace the `puts` and just call the `text` method on the object:

```ruby
translation.text #=> "Salve mundi!"
```